bin_native/test_skia.re

```diff
@@ -9,35 +9,35 @@ let imageInfo = Sk.ImageInfo.make(800, 600, N32, Premul, None);
 let surface = Sk.Surface.makeRaster(imageInfo, None);
 let canvas = Sk.Surface.getCanvas(surface);
 
-let bgColor = Sk.Color.make(255, 100, 0, 200);
+let bgColor = Sk.Color.makeARGB(255, 100, 0, 200);
```

Comment:
@manuhornung Changed this naming to make more clear which param is what (as CanvasKit has different ordering, I was going crazy 😄 )